CHANGELOG.md

```diff
@@ -21,17 +21,13 @@ modified Semantic Versioning scheme. See the "Versioning scheme" section of the
 
 ### Security
 
-## [2.0.8] - 2024-07-25
```

Comment:
I think we should leave the changelog entries alone and just revert the composer changes. Probably should add a new entry to the changelog instead.